projects/media-viewer/src/lib/media-viewer/toolbar/search-bar/search-bar.component.spec.ts

```diff
@@ -36,12 +36,30 @@ describe('SearchBarComponent', () => {
   });
 
   it('should not show searchbar', () => {
+    component.searchBarHidden.next(true);
+    fixture.detectChanges();
+
     const searchbar = nativeElement.querySelector('.findbar');
 
     expect(searchbar.className).toContain('hidden');
   });
 
+  it('should show searchbar after f3 keypress', () => {
+    component.searchBarHidden.next(true);
+    fixture.detectChanges();
+
+    const searchbar = nativeElement.querySelector('.findbar');
+    expect(searchbar.className).toContain('hidden');
+
+    const event = new KeyboardEvent('keydown', { 'code': 'F3' });
+    window.dispatchEvent(event);
+    fixture.detectChanges();
+
+    expect(searchbar.className).not.toContain('hidden');
+  });
+
   it('should emit search next event', () => {
+    component.searchBarHidden.next(false);
     const searchSpy = spyOn(component.searchEvents, 'next');
     component.searchText = 'searchTerm';
     const searchNextButton = nativeElement.querySelector('button[id=findNext]');
```

Comment:
"component.findNext.nativeElement" would be more consistent with line 25's change